README.md

```diff
@@ -27,14 +27,16 @@ made user-friendly for everyone to use. It allows you to execute network related
 For this installation we will assume that we are working on AlmaLinux 8 or 9. Warning: This guide does not cover any security hardening or rate limiting.
 Note: These steps also work with AlmaLinux 9, but it will install PHP 8 instead of 7.
 
-1. Install the required network tools: `dnf install mtr traceroute -y`.
+1. Install the required network tools: `dnf install mtr traceroute iperf3 -y`.
```

Comment:
Can you put the iPerf installation guide in a separate section? As mentioned earlier, this makes the installation more complex and not everybody runs an iPerf server. If you do run it, usually you do this on a separate server due to congestion (not exactly ideal for a looking glass).

Reply:
Yeah, no problem. By the way, would it be good if I comment out iPerf3-related configurations in `docker-compose.yml`? This way, when the user installs via Docker, it doesn't come with iPerf3 by default, since usually it should be deployed separately.

Reply:
Yes, good idea! We need to keep as much as possible optional, and very easy too.